### ISSUE TITLE
Smart Dashboard loses data in Test Mode

### ISSUE THREAD

**Issue by Kevin-OConnor** [wpilibsuite/SmartDashboard#73]
_From @KenBaron on January 23, 2017 1:55_

The following thread on Chief Delphi describes very well the problem we encountered today. Basically, if the live-window layout is saved in the Smartdashboard. The live window groups will be empty on the next reboot.

https://www.chiefdelphi.com/forums/showthread.php?t=145064

_Copied from original issue: wpilibsuite/allwpilib#471_

**Comment by NelsonBros** [comment]
I posted this on Chief Delphi last year 

https://www.chiefdelphi.com/forums/showthread.php?t=145064

and again this year

https://www.chiefdelphi.com/forums/showthread.php?t=157279   This one includes some screen shots.

And since then another post

https://www.chiefdelphi.com/forums/showthread.php?t=157433

I am a big fan of Robot Builder and Test Mode.  I will keep using them.  But my life would be better if this was fixed.
